docs/development/pigs/pig-001.rst

```diff
@@ -0,0 +1,120 @@
+.. include:: ../../references.txt
+
+.. _pig-001:
+
+**********************************
+PIG 1 - PIG purpose and guidelines
+**********************************
+
+* Author: Christoph Deil
+* Created: December 20, 2017
+* Status: draft
+
+What is a PIG?
+==============
+
```

Comment:
*This article is about the design document . For other uses, see `Pig (disambiguation)<https://en.wikipedia.org/wiki/Pig_(disambiguation)>`_.*

Reply:
Added. :-)